packages/eslint-config/package.json

```diff
@@ -9,9 +9,14 @@
     "eslint": "^8.40.0",
     "eslint-config-next": "13.4.12",
     "eslint-config-prettier": "^8.8.0",
+    "eslint-config-turbo": "^1.9.3",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
-    "eslint-config-turbo": "^1.9.3",
     "typescript": "^5.1.6"
+  },
+  "devDependencies": {
+    "eslint-plugin-import": "^2.28.1",
```

Comment:
This one is my enemy

Reply:
IMHO it's a pretty good rule to have I feel, but like please lmk if I need to remove it 👀

Reply:
I'd remove it, I've spent years battling it and making it work across everything that I attempt to import and have come to the conclusion that it's not worth it. 

The things that the prettier plugin missed, typescript catches and so it's not worth the overhead.

I do appreciate the initiative though!

Reply:
Yeah, I can understand. That plugin is a real pain though xD

Removed it 👀